app/assets/javascripts/application.js

```diff
@@ -9,7 +9,6 @@
 //= require ./jqplot.highlighter.min.js
 //= require ./jqplot.pieRenderer.min.js
 //= require jquery_nested_form
-//= require bootstrap.min
```

Comment:
😮

Reply:
interestingly, removing this causes no problem on my end - I do get an endless 404 when it's kept in

Reply:
I've added that now back in